src/compiler/types.ts

```diff
@@ -3358,7 +3364,7 @@ namespace ts {
     /* @internal */
     export interface CommandLineOptionOfListType extends CommandLineOptionBase {
         type: "list";
-        element: CommandLineOptionOfCustomType | CommandLineOptionOfPrimitiveType;
+        element: CommandLineOptionOfCustomType | CommandLineOptionOfPrimitiveType | TsConfigOnlyOption;

```

Comment:
It's how everything here is named. Note that `CommandLineOptionBase` has an `isTSConfigOnly` property, which you would think would have to be false by definition